packages/create-redwood-app/templates/js/web/jsconfig.json

```diff
@@ -43,7 +43,7 @@
   },
   "include": [
     "src",
-    "config",
```

Comment:
`config` is also used for storing the Tailwind config and PostCSS config if those are setup, so we should probably keep this one around (or update those setup commands to add this to the jsconfig/tsconfig files)

Reply:
> (or update those setup commands to add this to the jsconfig/tsconfig files)

If i had my way, that would be the best (as it would keep anything as minimal as possible, not adding those to your tsconfig unless actually needed. Going down that route, the `cli-storybook-vite` setup command would do what this PR does.

However, this is the easiest fix and as i currently don't have the resources to change and test all of these setup scripts, this will have to suffice for the moment ;)